packages/forma-36-react-components/src/components/IconButton/IconButton.tsx

```diff
@@ -72,38 +83,24 @@ export const IconButton = ({
     ...otherProps,
   };
 
-  const content = (
-    <TabFocusTrap className={styles.IconButton__inner}>
-      <Icon
-        {...iconProps}
-        className={cn(styles.IconButton__icon, iconProps.className)}
-      />
-      {label && <span className={styles.IconButton__label}>{label}</span>}
-      {withDropdown && (
-        <Icon
-          icon="ChevronDown"
-          color="secondary"
-          className={styles.IconButton__dropdown}
-        />
-      )}
-    </TabFocusTrap>
-  );
-
-  if (href) {
-    if (disabled) {
-      return <a {...elementProps}>{content}</a>;
-    }
-    return (
-      <a {...elementProps} href={href}>
-        content
-      </a>
-    );
-  }
-
   return (
-    <button {...elementProps} type="button" disabled={disabled}>
-      {content}
-    </button>
+    <Button
+      {...elementProps}
+      className={classNames}
+      innerWrapperClassName={styles[`IconButton--${size}`]}
```

Comment:
Is there a big reason we do funky things like this instead of just composing an `<Icon />` inside `<Button />`? 🙂

Reply:
`IconButton` was basically duplicate of `Button` they both have a very similar api with `Icon` inside. The new iconButton styles are basically "muted" button.

In order to use the button and adjust some padding I had to do the `innerWrapperClassName` magic, as our button dimensions defined on a child wrapper and not the main wrapper. This led me to refactor and simplify the button component but for V4 https://github.com/contentful/forma-36/pull/779